x/pylons/keeper/msg_server_account_test.go

```diff
@@ -97,6 +102,16 @@ func (suite *IntegrationTestSuite) TestUpdateAccount() {
 			request: &types.MsgUpdateAccount{Creator: addr[0], Username: "testUserUpdated"},
 			err:     types.ErrDuplicateUsername,
 		},
+		{
+			desc:    "Account not created",
+			request: &types.MsgUpdateAccount{Creator: addr[1], Username: "testUserUpdated1"},
+			err:     sdkerrors.ErrInvalidRequest,
+		},
+		{
+			desc:    "Invalid payfee",
```

Comment:
Description probably would read "Invalid fee"

Reply:
Thank you very much! I have fix in commit https://github.com/Pylons-tech/pylons/pull/870/commits/8aa682a4948fce1f64f88c04f253cbb7e3b4e2f1